pkg/generate/app/pipeline.go

```diff
@@ -137,6 +141,78 @@ func (pb *pipelineBuilder) NewBuildPipeline(from string, resolvedMatch *Componen
 	}, nil
 }
 
+/*
+// NewScratchBuildPipeline creates a pipeline without an input image and with an optional source repository.
```

Comment:
Should have been deleted, fixed now.

On Thu, Dec 10, 2015 at 2:15 PM, Cesar Wong <notifications@github.com>
wrote:

> In pkg/generate/app/pipeline.go
> <https://github.com/openshift/origin/pull/6253#discussion_r47270457>:
>
> > @@ -137,6 +141,78 @@ func (pb *pipelineBuilder) NewBuildPipeline(from string, resolvedMatch *Componen
> >  	}, nil
> >  }
> >
> > +/*
> > +// NewScratchBuildPipeline creates a pipeline without an input image and with an optional source repository.
>
> Is this here for doc purposes?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/6253/files#r47270457>.
>